pkg/services/screenshot/screenshot.go

```diff
@@ -92,6 +92,17 @@ func (s *HeadlessScreenshotService) Take(ctx context.Context, opts ScreenshotOpt
 		return nil, err
 	}
 
+	// If To and From have not been set then use the defaults for the panel
+	if opts.To == "" || opts.From == "" {
+		r, err := dashboard.GetTimeRange()
+		if err != nil {
+			s.instrumentError(err)
+			return nil, err
+		}
```

Comment:
`SetDefaults()` will set `From` and `To` if we miss setting them here, I don't think we need to return here since we can recover from this error by falling back to those default values.